app/models/manageiq/providers/cloud_manager.rb

```diff
@@ -64,5 +65,17 @@ def stop_event_monitor_queue_on_credential_change
     def supports_cloud_tenants?
       false
     end
+
+    def sync_cloud_tenants_with_tenants
```

Comment:
- this method it stored in MiqQueue, and it was inserted into MiqQueue in post refresh hook https://github.com/ManageIQ/manageiq/pull/10025/files#diff-f1cbe752863dde647be989fb5e615e2dR37

- yes it will do more in future, there will be syncing of CloudTenants to Tenants.

- I guess yes, because although yes there is method for syncing one CloudManager to Tenant(as root of openstack tree ) and also there will be methods for syncing whole tree of CloudTenants to Tenants - so this the reason of plural

ok ?